src/Lykke.Job.Messages/QueueConsumers/EmailQueueConsumer.cs

```diff
@@ -504,6 +513,29 @@ await _log.WriteInfoAsync
             await _smtpEmailSender.SendEmailAsync(result.PartnerId, result.EmailAddress, msg);
         }
 
+        private async Task HandleSwiftCashoutProcessedCypEmailAsync(SendEmailData<SwiftCashoutProcessedCypData> result)
+        {
+            await _log.WriteInfoAsync(nameof(EmailQueueConsumer), nameof(HandleSwiftCashoutProcessedCypEmailAsync), null, $"DT: {DateTime.UtcNow.ToIsoDateTime()}" +

```

Comment:
(Here and further) Please consider using sync log methods.
For example, https://github.com/LykkeCity/Lykke.Job.TransactionHandler/blob/dev/src/Lykke.Job.TransactionHandler/Handlers/EthereumCoreCommandHandler.cs#L72
https://github.com/LykkeCity/Lykke.Job.TransactionHandler/blob/dev/src/Lykke.Job.TransactionHandler/Handlers/EthereumCoreCommandHandler.cs#L115
And I doubt we need explicit datetime in logs.